### DISCUSSION TITLE
Question: How to rebind accept suggestion to Tab instead of Right?

### DISCUSSION THREAD

**Discussion by thecouchcoder** [cmderdev/cmder#2953] [Q&A] [answered]
Using Right to accept a suggestion causes excessive hand movement.  I'd like to rebind this to Tab.  I've looked through the docs and the settings and can't seem to figure out how to do it (I swear it used to be tab but maybe I'm misremembering??).  Any help is appreciated.

**Comment by chrisant996** [comment, marked as answer]
Turn it off instead.  Tab is for completion.  You do not really want Tab for suggestions instead. 

Search questions in the Clink repo for more info, or read the Clink docs.

**Reply by thecouchcoder** [reply]
Ah I didn't realize there was a difference between Completion and Suggestions.  Thanks!
